packages/core/src/log4js.classes.ts

```diff
@@ -37,6 +37,7 @@ export class Log4jsLogger implements LoggerService {
   error(message: any, trace?: string, context?: string) {
     this.updateContext(context);
     this.logger.error(message);
+    this.printStackTrace(trace);
```

Comment:
Error msg & error in the same line will be better.

like this,
https://github.com/log4js-node/log4js-node/blob/f8d46a939279c0ab4efc8bb5f0478c4b0949a4cf/test/tap/logging-test.js#L33